src/components/_inputs/js/InputNumber.js

```diff
@@ -0,0 +1,28 @@
+export default {
+  name: 'InputNumber',
+  props: {
+    name: {
+      type: String,
+      required: true
+    },
+    minimum: {
+      type: Number,
+      default: -99999999
+    },
+    maximum: {
+      type: Number,
+      default: 99999999
+    },
+    default: {
+      type: Number
+    }
+  },
+  data: () => {
+    return {
+      value: null
+    }
+  },
+  created () {
+    this.value = this.default
```

Comment:
Não deveria funcionar com `v-model`?

Reply:
Isso na verdade mudou, e agora vem de `this.schema.default`, nesse caso esse valor deverá ser carregado somente se usuário estiver criando as configurações, certo?

Reply:
De qualquer forma deveria suportar `v-model`, o `this.schema.default` é só o valor inicial, isto se não houver um outro valor já salvo vindo do `data`/`hidden_data`

Reply:
Mas naturalmente você precisa permitir que o componente parente controle o valor do campo, sendo um input o ideal é poder fazer isto com `v-model`...